core/state_processor_test.go

```diff
@@ -0,0 +1,834 @@
+package core
+
+import (
+	"math/big"
+	"testing"
+
```

Comment:
Minor nit, we could keep these stdlib imports together and then gofmt at the end.